config/mocha.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+
+  env: {
+    'mocha': true,
+  },
+
+  rules: {
+    'mocha/handle-done-callback': 'error',
+    'mocha/max-top-level-suites': [ 'error', { limit: 1 } ],
+    'mocha/no-async-describe': 'error',
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-global-tests': 'error',
+    'mocha/no-hooks': 'error',
+    'mocha/no-hooks-for-single-case': 'error',
+    'mocha/no-identical-title': 'error',
+    'mocha/no-mocha-arrows': 'error',
```

Comment:
I'm curious about this change - the only reason given by the Mocha team for this recommendation is to allow access to the Mocha context, which we don't use. We do use the local function context within tests often enough though, which arrow functions are especially handy for.

Do you think we'll find a use for the mocha context more going forward? I'd prefer to use arrow functions for all tests personally. Or that we switch to Jest, but that's another matter...

Reply:
> I'd prefer to use arrow functions for all tests personally. Or that we switch to Jest [...]

I would like to switch to Jest as well. I currently think that not using arrow functions makes the most sense for a _standard_ (we _could disable it in the extension? but I'd rather not) since it could be confusing to have the Mocha context work sometimes and not others.

Reply:
On switching to Jest, all the TypeScript projects will likely include that, since it does have better TypeScript support.

Reply:
Ok, fair enough. I didn't think we used the Mocha context anywhere, but we do in a few places it seems.